### PR TITLE
Restructure forked package

### PR DESCRIPTION
**Pull request by frinzekt**
This includes changes:
- github action for flake8 and publishing to pypi
- modification of the readme files
- changed the JWT package to package that is being updated

Note: I am not exactly sure if the version tagging syncs with pypi

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2016-2017 Fang Li
+Copyright 2020 - Lotterywest GoRillas Team
```
Comment (ignaciocabeza) [inline LICENSE]: We are in 2021 😬

File: django_saml2_auth_lw/views.py [hunk 2]
```diff
@@ -69,9 +62,12 @@ def get_reverse(objs):
     for obj in objs:
         try:
             return reverse(obj)
-        except:
+        except Exception:
```
Comment (ignaciocabeza) [inline django_saml2_auth_lw/views.py:65]: I think if you're not doing anything with the exception, this is not necessary. Is this part of some autoformatting tool?

File: setup.py [hunk 3]
```diff
@@ -10,21 +10,21 @@
...
+    long_description_content_type='text/markdown',
+    use_scm_version=True,
+    setup_requires=['setuptools_scm'],
 
-    url='https://github.com/fangli/django-saml2-auth',
+    url='https://github.com/Lotterywest/django-saml2-auth-lw',
 
-    author='Fang Li',
-    author_email='surivlee+djsaml2auth@gmail.com',
+    author='Lotterywest GoRillas Team',
```
Comment (ignaciocabeza) [inline setup.py]: We probably have to keep the original author

```
author='Fang Li, Lotterywest GoRillas Team',
author_email='surivlee+djsaml2auth@gmail.com, Gorillas@lotterywest.wa.gov.au',
```

File: setup.py [hunk 4]
```diff
@@ -10,21 +10,21 @@
...
 
 # Get the long description from the README file
-with open(path.join(here, 'README.rst'), encoding='utf-8') as f:
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
 setup(
-    name='django_saml2_auth',
-
-    version='2.2.1',
```
Comment (ignaciocabeza) [inline setup.py:19]: Put `2.2.2` here

File: README.md [hunk 5]
```diff
@@ -0,0 +1,260 @@
+# Django SAML2 Authentication Made Easy
+
+By Lotterywest GoRillas Team
+
+Version: Use 1.1.4 for Django <=1.9, 2.x.x for Django >= 1.9, Latest supported django version is 2.1
```
Comment (ignaciocabeza) [inline README.md]: lastest support is django 3.x

File: README.md [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+
+This project aims to provide a dead simple way to integrate SAML2
+Authentication into your Django powered app. Try it now, and get rid of the
+complicated configuration of SAML.
+
+Any SAML2 based SSO(Single-Sign-On) identity provider with dynamic metadata
+configuration is supported by this Django plugin, for example Okta.
+
```
Comment (ignaciocabeza) [inline README.md:13]: Probably explain why we forked here

File: django_saml2_auth_lw/views.py (hunk 2 above)
Comment (frinzekt) [reply django_saml2_auth_lw/views.py:65]: yes

File: README.md (hunk 6 above)
Comment (frinzekt) [reply README.md:13]: yeap

File: setup.py (hunk 4 above)
Comment (frinzekt) [reply setup.py:19]: I made it use autoversioning
```
    use_scm_version=True,
    setup_requires=['setuptools_scm'],

```
I think this is the right way to tag in the git history, then build will automatically find what version it is.

File: django_saml2_auth_lw/views.py (hunk 2 above)
Comment (frinzekt) [reply django_saml2_auth_lw/views.py:65]: https://www.flake8rules.com/rules/E722.html

**Comment by frinzekt** [comment]
Hold on a sec, im testing removing the package if it worked

**Review by ignaciocabeza — APPROVED** [review]

**Comment by ignaciocabeza** [comment]
All good for now. I will merge... we can do extra fixesin the future

**Pull request merged**